packages/editor/src/components/blocks/button/Button.tsx

```diff
@@ -34,13 +34,26 @@ export const useButtonComponent = () => {
       { label: t('components.button.type.danger'), value: 'DANGER' }
     ] as const;
 
+    const confirmDialogSeverity: FieldOption<ConfirmDialogSeverity>[] = [
+      { label: t('components.button.confirmSeverity.info'), value: 'INFO' },
+      { label: t('components.button.confirmSeverity.success'), value: 'SUCCESS' },
+      { label: t('components.button.confirmSeverity.warn'), value: 'WARN' },
+      { label: t('components.button.confirmSeverity.error'), value: 'ERROR' }
+    ] as const;
+
     const defaultButtonProps: Button = {
       name: t('property.action'),
       action: '',
       variant: 'PRIMARY',
       type: 'BUTTON',
       icon: '',
       processOnlySelf: false,
+      confirmDialog: false,
+      confirmMessage: t('property.confirmDialogMessage'),
+      confirmHeader: t('property.confirmDialogHeader'),
+      confirmSeverity: 'WARN',
+      confirmCancelValue: t('property.cancel'),
+      confirmConfirmValue: t('property.ok'),
```

Comment:
Also as this are not labels for property inputs but default values, I think these here should live under another key prefix, e.g. button.confirm.cancel...